re2/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.0)
 project(re2 CXX C)
 
+add_definitions(-Wno-narrowing)
```

Comment:
gcc 5.x is the latest one ;-) https://gcc.gnu.org/ml/gcc/2014-10/msg00040.html

And there are some changes, for example CFLAGS ***-std=gnu11*** https://gcc.gnu.org/gcc-5/changes.html